src/views/shared/alert/alert.styles.ts

```diff
@@ -1,11 +1,16 @@
 import { createUseStyles } from "react-jss";
-import { AlertVariant } from "./alert.view";
+import { AlertVariant } from "src/views/shared/alert/alert.view";
+import { Theme } from "src/styles/theme";
 
-const useAlertStyles = createUseStyles((theme) => ({
+const useAlertStyles = createUseStyles<
+  "root" | "icon" | "messageWrapper" | "message" | "helpButton",
+  { variant: AlertVariant },
+  Theme
+>((theme: Theme) => ({
   root: ({ variant }) => ({
     width: "100%",
     fontSize: theme.spacing(1.75),
-    background: variant === AlertVariant.LIGHT ? theme.palette.white : theme.palette.black,
+    background: variant === "light" ? theme.palette.white : theme.palette.black,
```

Comment:
why stop using the enum?

Reply:
Because in TS we can use a string literal union and I'd say it's a more common solution for these cases.

Reply:
Oh okay. I guess I'm biased from what I'm used to, but enums feel like they are easier to change as it's only in one place